<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab12</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="importTab">
       <attribute name="title">
        <string>Import</string>
       </attribute>
       <widget class="QWidget" name="i_main" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1131</width>
          <height>531</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>791</width>
          <height>531</height>
         </size>
        </property>
        <widget class="QWidget" name="r_i_main" native="true">
         <property name="geometry">
          <rect>
           <x>399</x>
           <y>9</y>
           <width>723</width>
           <height>513</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget">
            <property name="columnCount">
             <number>5</number>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>134</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>10</number>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Date of a flight</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Time of a flight</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Time in air</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Breakfast aviability</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Airport of destination</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="l_i_main" native="true">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>381</width>
           <height>513</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>384</width>
           <height>513</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="7" column="0">
           <widget class="QLineEdit" name="airportEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QDateEdit" name="flightDate">
            <property name="dateTime">
             <datetime>
              <hour>0</hour>
              <minute>0</minute>
              <second>0</second>
              <year>2021</year>
              <month>1</month>
              <day>1</day>
             </datetime>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="airLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>13</height>
             </size>
            </property>
            <property name="text">
             <string>Time in air</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QPushButton" name="binImportButton">
            <property name="text">
             <string>Import from a binary file</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="dateLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Date of a flight</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QTimeEdit" name="airTimeEdit">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QTimeEdit" name="flightTimeEdit">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="airportLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Airport of destination</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QPushButton" name="addButton">
            <property name="palette">
             <palette>
              <active>
               <colorrole role="ButtonText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="ButtonText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="ButtonText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>120</red>
                  <green>120</green>
                  <blue>120</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>193</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="10" column="0">
           <widget class="QPushButton" name="txtImportButton">
            <property name="text">
             <string>Import from a text file</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="timeLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Time of a flight</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QPushButton" name="csvImportButton">
            <property name="text">
             <string>Import from a CSV file</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="deleteTab">
       <attribute name="title">
        <string>Delete and Sort</string>
       </attribute>
       <widget class="QWidget" name="d_main" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1131</width>
          <height>531</height>
         </rect>
        </property>
        <widget class="QWidget" name="l_d_main" native="true">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>381</width>
           <height>513</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>381</width>
           <height>481</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteAllButton">
            <property name="text">
             <string>Delete all</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Order</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="ascRadio">
              <property name="text">
               <string>Ascending</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="desRadio">
              <property name="text">
               <string>Descending</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="dfSortButton">
            <property name="text">
             <string>Sort by the date of a flight</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tfSortButton">
            <property name="text">
             <string>Sort by the time of a flight</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="taSortButton">
            <property name="text">
             <string>Sort by the time in air</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="baSortButton">
            <property name="text">
             <string>Sort by breakfast aviability</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="adSortButton">
            <property name="text">
             <string>Sort by the airport of destination</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="r_d_main" native="true">
         <property name="geometry">
          <rect>
           <x>399</x>
           <y>9</y>
           <width>723</width>
           <height>513</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>311</width>
           <height>491</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget_2">
            <property name="columnCount">
             <number>5</number>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>134</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>10</number>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Date of a flight</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Time of a flight</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Time in air</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Breakfast aviability</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Airport of destination</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="exportTab">
       <attribute name="title">
        <string>Export</string>
       </attribute>
       <widget class="QWidget" name="e_main" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1131</width>
          <height>531</height>
         </rect>
        </property>
        <widget class="QWidget" name="l_e_main" native="true">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>381</width>
           <height>513</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>381</width>
           <height>513</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="plainExportButton">
            <property name="text">
             <string>Export as a plain text</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="tableExportButton">
            <property name="text">
             <string>Export as a table</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="csvExportButton">
            <property name="text">
             <string>Export as a CSV file</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="binExportButton">
            <property name="text">
             <string>Export as a binary file</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="r_e_main" native="true">
         <property name="geometry">
          <rect>
           <x>399</x>
           <y>9</y>
           <width>723</width>
           <height>513</height>
          </rect>
         </property>
         <widget class="QTableWidget" name="tableWidget_3">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>9</y>
            <width>705</width>
            <height>495</height>
           </rect>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>134</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>10</number>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Date of a flight</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Time of a flight</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Time in air</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Breakfast aviability</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Airport of destination</string>
           </property>
          </column>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
